documentation/content/oauth/providers/google.md

```diff
@@ -27,19 +27,21 @@ const google: (
 		clientSecret: string;
 		redirectUri: string;
 		scope?: string[];
+		access_type?: "online" | "offline";
 	}
 ) => OAuthProvider<GoogleUser, GoogleTokens>;
 ```
 
 #### Parameter
 
-| name                 | type                                 | description                    | optional |
-| -------------------- | ------------------------------------ | ------------------------------ | :------: |
-| auth                 | [`Auth`](/reference/lucia-auth/auth) | Lucia instance                 |          |
-| configs.clientId     | `string`                             | Google OAuth app client id     |          |
-| configs.clientSecret | `string`                             | Google OAuth app client secret |          |
-| configs.redirectUri  | `string`                             | an authorized redirect URI     |          |
-| configs.scope        | `string[]`                           | an array of scopes             |    ✓     |
+| name                 | type                                 | description                                                                     | optional |
+| -------------------- | ------------------------------------ | ------------------------------------------------------------------------------- | :------: |
+| auth                 | [`Auth`](/reference/lucia-auth/auth) | Lucia instance                                                                  |          |
+| configs.clientId     | `string`                             | Google OAuth app client id                                                      |          |
+| configs.clientSecret | `string`                             | Google OAuth app client secret                                                  |          |
+| configs.redirectUri  | `string`                             | an authorized redirect URI                                                      |          |
+| configs.scope        | `string[]`                           | an array of scopes                                                              |    ✓     |
+| configs.access_type  | `string`                             | Google OAuth Access type ("online" (default) or "offline" (gets refresh_token)) |    ✓     |
```

Comment:
The type should be `"online" \| "offline"` (include the escape), and I think we should create a new `default` column.